Replaced hardcoded strings with resource strings - merge to master

MariskaIJpelaar opened the pull request:
Converted (hopefully) all hardcoded strings to resource strings.
Fixed small resource error (mipmap files)
Requires review before merging with master

TODO:
  - Spelling and consistency check
  - Add Dutch as a language
  - make 'settings' so the language can be changed

MariskaIJpelaar committed:
Merged from master
Converted (hopefully) all hardcoded strings to resource strings.
Fixed small resource error (mipmap files)
Requires review before merging with master

TODO:
  - Spelling and consistency check
  - Add Dutch as a language
  - make 'settings' so the language can be changed

MariskaIJpelaar commented on app/src/main/java/com/sebastiaan/xenopelthis/ui/main/SectionsPagerAdapter.java:
```diff
@@ -1,21 +1,31 @@
 package com.sebastiaan.xenopelthis.ui.main;
```
Could you review this file please?

MariskaIJpelaar commented on app/src/main/java/com/sebastiaan/xenopelthis/ui/main/activity/main/MainActivity.java:
```diff
@@ -32,7 +32,7 @@ protected void onCreate(Bundle savedInstanceState) {
         model = ViewModelProviders.of(this).get(MainViewModel.class);
 
-        SectionsPagerAdapter sectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager());
+        SectionsPagerAdapter sectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager(), this);
```
Could you review this file please?

MariskaIJpelaar commented on app/src/main/java/com/sebastiaan/xenopelthis/ui/product/view/dialog/OverrideDialog.java:
```diff
@@ -1,6 +1,7 @@
 package com.sebastiaan.xenopelthis.ui.product.view.dialog;
```
Could you review this file, please?

MariskaIJpelaar commented on app/src/main/java/com/sebastiaan/xenopelthis/ui/supplier/view/dialog/OverrideDialog.java:
```diff
@@ -1,6 +1,7 @@
 package com.sebastiaan.xenopelthis.ui.supplier.view.dialog;
```
Could you review this file please?

MariskaIJpelaar commented:
Added some comments on the files I want to have reviewed specifically. @Sebastiaan-Alvarez-Rodriguez

Sebastiaan-Alvarez-Rodriguez reviewed (approved):
Your current way of fetching resource strings is fine by me, as long as it works well.
The way I know of is `getResources().getString(R.string.mystring);`. 
Maybe worth investigating.

MariskaIJpelaar committed:
Added dutch strings.

TODO:
  - Let the app take the default language of the device
  - User should be able to change language in settings

MariskaIJpelaar committed:
Relocated the dutch-values folder.
Android Studio now takes the default device language as the app language.

TODO:
  - Add settings
